### PR TITLE
Fluentbit

### PR DESCRIPTION
**Pull request by ben851**
## What happens when your PR merges?
Staging will move from fluentd to fluentbit for log aggregation.

## What are you changing?
- [X] Changing kubernetes configuration

## Provide some background on the changes
Fluentd breaks in v1.24 of kubernetes - since it is so fragile, I've moved us to fluent bit which seems to be better supported for EKS.

## Checklist if making changes to Kubernetes:
- [X] I know how to get kubectl credentials in case it catches on fire

### CODE REVIEW

File: env/scratch/fluentbit.yaml [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+  name: fluent-bit-config
+  namespace: amazon-cloudwatch
+  labels:
+    k8s-app: fluent-bit
+data:
+  fluent-bit.conf: |
+    [SERVICE]
+        Flush                     5
+        Grace                     30
+        Log_Level                 info
```
Comment (jimleroyer) [inline env/scratch/fluentbit.yaml:63]: This is for the logging level of fluentbit itself I assume?

File: env/scratch/fluentbit.yaml [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+    [OUTPUT]
+        Name                cloudwatch
+        Match               application.*
+        region              ${AWS_REGION}
+        log_group_name      /aws/containerinsights/${CLUSTER_NAME}/application
+        log_stream_name     $(tag[4])
+        auto_create_group   true
+        extra_user_agent    container-insights
+
+  dataplane-log.conf: |
```
Comment (jimleroyer) [inline env/scratch/fluentbit.yaml:139]: Why is it called data plane?

**Review by jimleroyer — APPROVED** [review]
Looks good, as far as I can tell. 😅

File: env/scratch/fluentbit.yaml (hunk 1 above)
Comment (ben851) [reply env/scratch/fluentbit.yaml:63]: Yeah the [SERVICE] section is the fluentbit config section

**Pull request merged**